app/controllers/passkit/api/v1/passes_controller.rb

```diff
@@ -9,12 +9,12 @@ def create
 
           if @generator && @payload[:collection_name].present?
             files = @generator.public_send(@payload[:collection_name]).collect do |collection_item|
-              Passkit::Factory.create_pass(@payload[:pass_class], collection_item)
+              Passkit::Factory.create_pass(@payload[:pass_class], generator: collection_item, site_id: params[:site_id])
```

Comment:
Could we have some method that can be redefined in child controller, e.g. `pass_attributes` instead of using hardcoded `site_id` in gem? Don’t want gem to know anything about sites.